config/routes.rb

```diff
@@ -70,15 +70,21 @@
 
     get 'ready', on: :collection
     resources :options
+    resources :comments
   end
 
-  resources :users
   resources :ratings, only: [:increase, :decrease] do
     member do
       post 'increase'
       post 'decrease'
     end
   end
+
+  resources :comments do
+    resources :comments
```

Comment:
This really doesn't look right.
Maybe let's keep `comments` as a top-level resource, and not nesting them anywhere? This should be possible, since all your comments have unique ids...